source/config-file/config-file-options.txt

```diff
@@ -59,6 +59,26 @@ You can configure the following settings in a configuration file:
      - Configure |compass| to not perform outgoing network operations other 
        than those to the database.
 
+Example 
+-------
+To view a sample |compass-short| configuration file, run the following command: 
+
+.. code-block:: sh
+
+     <path-to-Compass-executable> --show-example-config
+
+If you do not already have a configuration file, you can create a configuration 
+file that uses the sample configuration settings:
+
+.. code-block:: sh 
+
+     <path-to-Compass-executable> --show-example-config > /etc/mongodb-compass.conf
+
+.. seealso:: 
+
+  To run |compass-short| with a configuration file, see 
+  :ref:`compass-connect-file-based`.
```

Comment:
Is this the right link? I’m not sure where this points but I assume it’s about connecting with a connections list file, not a configuration file

Reply:
Sorry about that — that's a misread on my end for what the section was discussing.